cms/templates/group_configurations.html

```diff
@@ -47,32 +49,45 @@ <h1 class="page-header">
 
   <div class="wrapper-content wrapper">
     <section class="content">
+
       <article class="content-primary" role="main">
-        <div class="wrapper-groups content-groups">
-          <h3 class="title">${_("Content Groups")}</h3>
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-        </div>
 
-      % if should_show_experiment_groups:
-        <div class="wrapper-groups experiment-groups">
-          <h3 class="title">${_("Experiment Group Configurations")}</h3>
-          % if experiment_group_configurations is None:
-            <div class="notice notice-incontext notice-moduledisabled">
-                <p class="copy">
-                    ${_("This module is disabled at the moment.")}
-                </p>
-            </div>
-          % else:
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-          % endif
-        </div>
-      % endif
+        % for config in all_group_configurations:
+          <div class="wrapper-groups content-groups ${config['scheme']}">
+            <h3 class="title">${config['name']}</h3>
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+          </div>
+        % endfor
+
+        % if should_show_experiment_groups:
+          <div class="wrapper-groups experiment-groups">
+            <h3 class="title">${_("Experiment Group Configurations")}</h3>
+            % if experiment_group_configurations is None:
+              <div class="notice notice-incontext notice-moduledisabled">
+                  <p class="copy">
+                      ${_("This module is disabled at the moment.")}
+                  </p>
+              </div>
+            % else:
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+            % endif
+          </div>
+        % endif
       </article>
       <aside class="content-supplementary" role="complementary">
+      % if should_show_enrollment_track:
+        <div class="bit">
+          <div class="enrollment-track-doc">
+            <h3 class="title-3">${_("Enrollment Track Content Groups")}</h3>
```

Comment:
@catong We can't yet edit on the course outline (at the unit level)-- we can only do it at the component level on the unit page. I'm reacting to this--

" In the course outline, you can designate some content as visible only to learners in a specific enrollment track."